components/bl-simpleModal-component/README.md

```diff
@@ -0,0 +1,54 @@
+# Simple Modal
+
+Simple Modal is a designer component of Backendless UI-Builder. This allows you to add a modal dialog box. You can add "Alert" to notify you about certain information. You can add "Prompt" to get information from the user. You can add "Confirm" to get confirmation at user for some actions.
+
+The component based on external [Simple Modal](https://mui.com/material-ui/react-dialog/).
+## Usage
+
+### Styles and Settings
+
+<dl>
+<dt>Title</dt>
+<dd>Allows write title for Simple Modal.</dd>
+<dt>Content</dt>
+<dd>Allows write content for Simple Modal</dd>
+<dt>Close Button Label</dt>
+<dd>Allows to write label for close button. Default value "Close"</dd>
+<dt>Type</dt>
+<dd>Allows select type of Simple Modal (Alert, Prompt, Confirm). Default value 'Alert'.</dd>
```

Comment:
Allows select type of Simple Modal ("alert", "prompt", "confirm"). Default value "alert".